### ISSUE TITLE
Search engines index the Bulgarian homepage

### ISSUE THREAD

**Issue by madewild** [ec-doris/kohesio-frontend#904]
See https://www.google.com/search?q=kohesio

**Comment by raphdom** [comment]
If there is no browser language detected, it is redirecting to the first language.

The easy test to detect it is do a curl directly in the website:

curl http://kohesio.ec.europa.eu

**Comment by madewild** [comment]
Is there a way to force english with a HTTP redirection from http://kohesio.ec.europa.eu/ to http://kohesio.ec.europa.eu/en/ ?

**Comment by raphdom** [comment]
We are doing this already, in the client browser, but this is the bot of google indexing the results

**Comment by madewild** [comment]
europa.eu is not having this problem so there must be a way...
I don't see anything in their robots.txt though

**Comment by raphdom** [comment]
yeah, there is a way, I put here a comment how I can reproduce the problem, now I know that we should control better the redirection, I mean:
- If we have a browser language, redirect to this language
- If we don't have a browser client, redirect to English (this part is missing)

**Comment by raphdom** [comment]
We still have the problem
![Screenshot 2024-09-03 at 16 23 14](https://github.com/user-attachments/assets/6058a2f1-66ba-470f-9df0-9fc6c3a2ad1f)

**Comment by madewild** [comment]
@ntorbev you mentioned you fixed it? please close if so
